oktaawscli/okta_auth.py

```diff
@@ -110,15 +112,15 @@ def verify_mfa(self, factors_list, state_token):
                                          from ~/.okta-aws")
                         break
                 else:
-                    print("%d: %s" % (index + 1, factor_name))
+                    self.logger.debug("%d: %s" % (index + 1, factor_name))
```

Comment:
This is being hidden behind `--debug`, but the user would regularly need to see this if they're not specifying a factor in their config.